test/e2e.test.js

```diff
@@ -49,8 +51,17 @@ test('demo app reports a vuln method when called', async (t) => {
 
   // expecting a call to homebase for the newest snapshot
   nock('http://localhost:8000')
+    .matchHeader('if-modified-since', (val) => {
+      // making sure we got a Date here since I'm not sure what else to test in the 1st request
```

Comment:
Good point. Should we send no date the first time? Or should we estimate the date of the embedded snapshot somehow? Homebass will happily reply to a request without an if-modified-since with its data. Maybe it shouldn't if it's outdated aaaah.

Reply:
taking this to slack; it's not a critical decision but I'm interested to hear opinions